src/series/getters.ts

```diff
@@ -31,48 +31,34 @@ export const getSeriesTitle = ($seriesPage: SeriesPage): string => {
   return $seriesPage("h2.heading").text().trim();
 };
 
-export const getSeriesAuthors = (
-  $seriesPage: SeriesPage
-): Series["authors"] => {
+export const getSeriesAuthors = ($seriesPage: SeriesPage): Series["authors"] => {
   const authorLinks = $seriesPage("dl.meta a[rel=author]");
-  const authors: Author[] = [];
 
-  if (
-    $seriesPage("dl.meta > dd:nth-of-type(1)").text().trim() === "Anonymous"
-  ) {
+  if ($seriesPage("dl.meta > dd:nth-of-type(1)").text().trim() === "Anonymous") {
     return [{ username: "Anonymous", pseud: "Anonymous", anonymous: true }];
   }
 
   if (authorLinks.length !== 0) {
-    authorLinks.each((i, element) => {
+    return authorLinks.map((_, element) => {
       const url = element.attribs.href;
       const [, username, pseud] = url.match(/users\/(.+)\/pseuds\/(.+)/)!;
-
-      authors.push({
+      return {
         username: username,
         pseud: decodeURI(pseud),
         anonymous: false,
-      });
-    });
+      } as Author
```

Comment:
We should figure out why you need "`as Author`" here. What happens if change line 42 to be

```ts
    return authorLinks.map((_, element): Author => {
 ```

Reply:
i'm pretty sure i don't need to. I'll check tomorrow